tutorials/cleaning-up-at-scale/index.md

```diff
@@ -0,0 +1,183 @@
+## Cleaning up Compute Engine instances at scale
+
+
+## Summary
+
+This tutorial offers a simple, serverless and scalable mechanism to automatically delete, or "[garbage collect][https://en.wikipedia.org/wiki/Garbage_collection_(computer_science)]", Compute Engine instances after a fixed time period.  Some use cases where this may be prove useful:
+
+* Developers or testers create one-off instances for testing features, but may not always remember to manually delete them.
+* Workflows that require dynamically spinning up a large number of GCE worker instances to perform a certain task.  While best practice is to have instances delete themselves after the task is complete, but ensuring that this always happens can be difficult if the task is distributed or some workers abort due to errors.
+
+This solution leverages the following GCP components: 
+
+*   **Compute Engine**
+*   **Cloud Scheduler**
+*   **Pub/Sub** 
+*   **Cloud Functions**
+
+![High-level overview of the solution](/images/overview.svg "High-level overview of the solution")
+
+**Figure 1: **High-level overview of the solution**
+
+### **How does it work**? 
+
+Each GCE instance in scope is assigned two labels: 
+
+*   **TTL** (Time to Live): which indicates (in minutes) after how much time this VM will not be needed and can be deleted.
+*   **ENV** : it indicates that the instance is part of the pool of VMs that can be checked regularly and can be deleted if TTL is reached. 
+
+The overall flow is the following: 
+
+
+1. We create a Cloud Scheduler cron job which will be triggered regularly (e.g each 5 minutes). The config of the Cloud scheduler contains the values of the label of the pool of VMs to target (e.g ENV= test). Optionally, it can also contain the name of the zone where the VMs to target are deployed. The format is the following: 
+
+		'{"zone":"us-west1-b", "label":"env=test"}'
+
+
+
+2. Once the cron job is triggered, Cloud Scheduler will push a message with the payload above to a Pub/Sub topic. 
+3. A Cloud Function is subscribed to the Pub/Sub topic. Each time triggered, it will perform the following: 
+    1.  Read the payload of the Pub/Sub message
+    2. Extract the **label** and the **zone**
```

Comment:
we only need to extract the "label", see my comment in line #34.

Reply:
Done

Reply:
LGTM